tests/units/common/test_sftp.py

```diff
@@ -23,15 +22,7 @@ def initiate_sftp_service():
     raises(DirectoryNotFoundException, initiate_sftp_service)
 
 
-def test_logs():
-    with capture_logs() as cap_logs:
+def test_error_raise():
+    with raises(Exception):
```

Comment:
Checking if the error is raised in __exit__.
It should be because we are returning False in __exit__:
https://github.com/cern-sis/workflows/blob/79606106d5e54b6d59f141ade34d2c885cd91303/dags/common/sftp_service.py#L55